package.json

```diff
@@ -101,6 +101,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-react": "^7.26.1",
     "jest": "^26.6.3",
+    "mockdate": "^3.0.5",
```

Comment:
I'm ok with adding mockdate. Adding a dev dependency isn't as bad as adding a regular dependency.

I believe that dev dependencies won't be included in production builds only local debug builds.